Src/Sarif.PatternMatcher/SearchSkimmer.cs

```diff
@@ -427,9 +428,10 @@ private void RunMatchExpression(FlexMatch binary64DecodedMatch, AnalyzeContext c
                                 FailureLevel.Error,
                                 exception: null,
                                 persistExceptionStack: false,
-                                messageFormat: SpamResources.ERR998_ValidatorReturnedIllegalValidationState,
```

Comment:
>ERR998_ValidatorReturnedIllegalValidationState [](start = 61, length = 46)

Oops, we used the wrong format string here and also failed to report args properly. For this error and also the illegal return value.